src/main/scala/com/wavesplatform/matcher/market/OrderBookActor.scala

```diff
@@ -60,6 +62,8 @@ class OrderBookActor(assetPair: AssetPair,
       onAddOrder(order)
     case cancel: CancelOrder =>
       onCancelOrder(cancel)
+    case OrderCleanup  =>
```

Comment:
Where is this message sent? Should be scheduled in a configurable period of time (default 5 min)